src/main/resources/public/js/components/html-head.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {Helmet} from 'react-helmet';
+
+export default class HtmlHead extends React.PureComponent {
+
+    state = {
+        hostName: null
+    };
+
+    componentDidMount = async () => {
+        let hostName = window.location.hostname;
+
+        let res = await fetch(`/img/favicons/${hostName}/favicon-16x16.png`);
```

Comment:
This will not work : `hostname` will be something like services.sictiam.fr whereas instances are keyed with something like sictiam